MASTER FUNCTIONS LIBRARY.vbs

```diff
@@ -4962,6 +4962,49 @@ function clear_line_of_text(row, start_column)
   EMWaitReady 0, 0
 end function
 
+function collect_script_usage_data(script_to_record, closing_message, reset_timer)
+'--- This function records script runs into the usage log and can be used for functionality runs, not just completed scripts.
+'~~~~~ script_to_record: this would be the 'name_of_script' for completed script runs, but can be adjusted if run on functionality instead of the completed script
+'~~~~~ closing_message: information about the script run should be passed through this variable, for completed script runs, this would be the closing message
+'===== Keywords: MAXIS, MMIS, script, statistics
```

Comment:
Line 4969: I'm assuming reset_timer is a Boolean, but can you add instructions for this parameter?

Reply:
I left it out because I'm not sure we need it - or if it makes sense to have it here at all.